src/main/scala/com.snowplowanalytics.snowplow.badrows/FailureDetails.scala

```diff
@@ -102,9 +102,7 @@ object FailureDetails {
   
   // SCHEMA VIOLATIONS
   
-  sealed trait EnrichmentStageIssue
```

Comment:
👍  Don't think we ever used it?

Reply:
It was used a lot in SCE in R118 https://github.com/snowplow/snowplow/blob/master/3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala#L76

I took the opportunity to remove it in [this PR](https://github.com/snowplow/snowplow/pull/4287) to always be as specific as we can be.